tiatoolbox/tools/patchextraction.py

```diff
@@ -92,7 +110,20 @@ def __init__(
         self.n = 0
         self.wsi = wsireader.get_wsireader(input_img=input_img)
         self.locations_df = None
+        self.coord_list = None
         self.stride = None
+        if input_mask is None:
+            self.mask = None
+        elif input_mask == "auto":
```

Comment:
can we specify `otsu` and `morphological` as are the options in tissue mask?
change `auto`  to `default` where tissue mask can use default mode.

Reply:
Yeah it's good idea to include `otsu` and `motphological` method as input. It's better than `auto` and if we are specifying the methods, the `auto` option is no longer valid. However, the default value for this argument should remain `None` as it's needed to distinguish between mask-based or non-mask-based patch extraction.

Reply:
resolved in 137e25c758490f665c967a81f0263c34f5d17677